core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -431,6 +431,11 @@ public String toString() {
     public PickResult pickSubchannel(PickSubchannelArgs args) {
       return PickResult.withNoResult();
     }
+
+    @Override
+    public String toString() {
```

Comment:
The default toString() should work fine, or toStringHelper.

Reply:
Done.